src/const.ts

```diff
@@ -0,0 +1,17 @@
+export const RLN_ABI = [
+  "function MEMBERSHIP_DEPOSIT() public view returns(uint256)",
+  "function register(uint256 pubkey) external payable",
+  "function withdraw(uint256 secret, uint256 _pubkeyIndex, address payable receiver) external",
+  "event MemberRegistered(uint256 pubkey, uint256 index)",
+  "event MemberWithdrawn(uint256 pubkey, uint256 index)",
+];
+
+export const DEV_CONTRACT = {
+  chainId: 5,
+  startBlock: 7109391,
+  address: "0x4252105670fe33d2947e8ead304969849e64f2a6",
+  abi: RLN_ABI,
+};
+
+export const DEFAULT_SIGNATURE_MESSAGE =
```

Comment:
I think the signature should be app defined.

I am not sure to see a use case where two apps would use the same zk credentials as one app could compromise the credentials for the other app.

What do you think @s1fr0?

Reply:
I though of it more as an example not an enforced thing.

Reply:
> I though of it more as an example not an enforced thing.

In this case I would not define it in this package but in the examples. If we provide this message it is guaranteed that users will use it.